doc/specs/#2046 - Command Palette.md

```diff
@@ -1,7 +1,7 @@
 ---
 author: Mike Griese @zadjii-msft
 created on: 2019-08-01
-last updated: 2020-04-13
+last updated: 2020-04-30
 issue id: 2046
 ---
 

```

Comment:
Hey @zadjii-msft, one line below this, there's some text that says "This spec is a draft". You should yank that!